infer/src/dune.common.in

```diff
@@ -15,6 +15,8 @@ let clang = is_yes "@BUILD_C_ANALYZERS@"
 
 let erlang = is_yes "@BUILD_ERLANG_ANALYZERS@"
 
+let windows = is_yes "@WINDOWS_BUILD@"
+
```

Comment:
this list isn't terribly well sorted to begin with because of `facebook` but this could still go lower in the file

Reply:
Done.